taskt/Core/Automation/Commands/Outlook Commands/SendOutlookEmailCommand.cs

```diff
@@ -1,93 +1,89 @@
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Outlook Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to send emails with outlook")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to send emails with your currenty logged in outlook account")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Outlook Commands")]
+    [Description("This command sends emails with attachments in Outlook.")]
+
     public class SendOutlookEmailCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Indicate Recipients (; delimited)")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the Email Addresses of the recipients in semicolon seperated values")]
-        [Attributes.PropertyAttributes.SampleUsage("test@test.com;test2@test.com")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Recipient(s)")]
+        [InputSpecification("Enter the email address(es) of the recipient(s).")]
+        [SampleUsage("test@test.com;test2@test.com || {vEmail1};{vEmail2} || {vEmails}")]
+        [Remarks("Multiple recipient email addresses should be delimited by a semicolon (;).")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_Recipients { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Attachment File Path (; delimited, Optional)")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the Filepath of the file you want attached.")]
-        [Attributes.PropertyAttributes.SampleUsage("c:sales reports\fy06q4.xlsx")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
-        public string v_Attachment { get; set; }
-
-        [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Provide Email Subject")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the subject you want sent in your Email")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Email Subject")]
+        [InputSpecification("Enter the subject you want sent in your Email")]
```

Comment:
You can simply write "Enter the subject of the email."
Punctuation is also missing.

Reply:
done